code/modules/vehicles/animal.dm

```diff
@@ -44,3 +46,156 @@
 	icon = 'icons/obj/saddle.dmi'
 	icon_state = "saddle"
 	w_class = ITEMSIZE_NORMAL
+
+/obj/vehicle/bike/threshbeast
+	name = "threshbeast"
+	desc = "Large herbivorous reptiles native to Moghes, the azkrazal or 'threshbeast' is commonly used as a mount, beast of burden, or convenient food source by Unathi. They are highly valued for their speed and strength, capable of running at 30-42 miles per hour at top speed. This one has been fitted with a saddle."
+	icon = 'icons/mob/npc/moghes_64.dmi'
+	icon_state = "threshbeast_s"
+	bike_icon = "threshbeast_s"
+	desc_info = "Click-drag yourself onto the animal to climb onto it.<br>\
+		- Click-drag it onto yourself to access its mounted storage.<br>"
+	pixel_x = -15
+	mob_offset_y = 10
+	kickstand = FALSE
+	on = TRUE
+	land_speed = 2
+	land_speed_careful = 3
+	space_speed = 0
+
+	health = 100
+
+	can_hover = FALSE
+	organic = TRUE
+
+	storage_type = /obj/item/storage/toolbox/bike_storage/saddle
+	corpse = /mob/living/simple_animal/threshbeast/saddle
+	spawns_with_key = FALSE
+	key_type = null
+
+/obj/vehicle/bike/threshbeast/setup_vehicle()
+	..()
+	on = TRUE
+	set_light(0)
+
+/obj/vehicle/bike/threshbeast/CtrlClick(var/mob/user)
+	return
+
+/obj/vehicle/bike/threshbeast/toggle_engine(var/mob/user)
+	return
+
+/obj/vehicle/bike/threshbeast/kickstand(var/mob/user)
+	return
+
+/obj/vehicle/bike/warmount
```

Comment:
this should be a better named type cause `warmount` is pretty generic and does not sound like it's a moghesian animal

Reply:
Renamed to `hegeranzi`